lib/elsa/supervisor.ex

```diff
@@ -72,13 +74,32 @@ defmodule Elsa.Supervisor do
   * `:config` - Optional. Consumer configuration options passed to `brod_consumer`.
 
 
+  ## Consumer Config
+
+  * `:topic` - Required. Topic to subscribe to.
+
+  * `:partition` - Required. Topic partition to subscribe to.
```

Comment:
This should be optional, we can subscribe to all the partitions if none are provided.